tests/system/small/test_multiindex.py

```diff
@@ -866,6 +886,17 @@ def test_column_multi_index_reorder_levels(scalars_df_index, scalars_pandas_df_i
     pandas.testing.assert_frame_equal(bf_result, pd_result)
 
 
+def test_multi_index_unstack(hockey_df, hockey_pandas_df):
+    bf_result = (
+        hockey_df.set_index(["team_name", "season", "position"]).unstack().to_pandas()
+    )
+    pd_result = hockey_pandas_df.set_index(
+        ["team_name", "season", "position"]
+    ).unstack()
```

Comment:
I think it would be worth testing for all levels: `{-3, -2, -1, 0, 1, 2}`

Reply:
I don't support levels param yet. Will add levels support to stack() and unstack() in a later pr.